test/e2e/apimachinery/webhook.go

```diff
@@ -123,6 +125,12 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		testCRDWebhook(f, testcrd.Crd, testcrd.DynamicClient)
 	})
 
+	It("Should not be able to prevent deleting validating-webhook-configurations", func() {
```

Comment:
The same is true for the mutating-webhook-configurations, could you expand the test?